### PR TITLE
#1736 new license type.

### PR DESCRIPTION
**Pull request by HryhoriiHevorkian**
Changed text for license from Apache 2.0 to AGPL 3.0.

### CODE REVIEW

**Comment by HryhoriiHevorkian** [comment]
@max402 please review.

File: opba-protocols/xs2a-protocol/src/main/java/de/adorsys/opba/protocol/xs2a/entrypoint/AccountStatementMapper.java [hunk 1]
```diff
@@ -1,17 +1,19 @@
 /*
- * Copyright 2018-2019 adorsys GmbH & Co KG
```
Comment (max402) [inline opba-protocols/xs2a-protocol/src/main/java/de/adorsys/opba/protocol/xs2a/entrypoint/AccountStatementMapper.java:2]: I don't know why this particular file contains license info. I'd remove it

File: backend.pmd-ruleset.xml [hunk 2]
```diff
@@ -1,22 +1,21 @@
-        <!--
```
Comment (max402) [inline backend.pmd-ruleset.xml:1]: I don't know why this particular file contains license info. I'd remove it

**Review by max402 — APPROVED** [review]

File: backend.pmd-ruleset.xml (hunk 2 above)
Comment (HryhoriiHevorkian) [reply backend.pmd-ruleset.xml:1]: Removed this.

File: pom.xml [hunk 3]
```diff
@@ -1,17 +1,20 @@
 <?xml version="1.0" encoding="UTF-8"?><!--
-  ~ Copyright 2018-2019 adorsys GmbH & Co KG
+<!--
```
Comment (max402) [inline pom.xml:2]: Error:      Non-parseable POM /home/runner/work/open-banking-gateway/open-banking-gateway/pom.xml: in comment after two dashes (--) next character must be > not \n (position: START_DOCUMENT seen <?xml version="1.0" encoding="UTF-8"?><!--\n<!--\n... @3:1)  @ line 3, column 1 -> [Help 2]

Comment (HryhoriiHevorkian) [reply pom.xml]: Done.

**Pull request merged**